Source/WebKit/UIProcess/Launcher/glib/XDGDBusProxy.h

```diff
@@ -27,25 +27,21 @@
 
 #if ENABLE(BUBBLEWRAP_SANDBOX)
 #include "ProcessLauncher.h"
-#include <wtf/Noncopyable.h>
-#include <wtf/TZoneMalloc.h>
 #include <wtf/Vector.h>
 #include <wtf/text/CString.h>
 #include <wtf/unix/UnixFileDescriptor.h>
 
 namespace WebKit {
 
 class XDGDBusProxy {
-    WTF_MAKE_TZONE_ALLOCATED(XDGDBusProxy);
-    WTF_MAKE_NONCOPYABLE(XDGDBusProxy);
```

Comment:
Hm, this part is going to conflict with https://github.com/WebKit/WebKit/pull/34133. We need to decide whether we really want to `dup()` the `m_syncFD` every time the ProcessLauncher gets copied (which can be done by writing a custom copy constructor and copy assignment operator), or whether we should avoid that and try making ProcessLauncher noncopyable instead.

Reply:
I've undone this, supposedly it won't conflict anymore?